pkg/generate/git/repository.go

```diff
@@ -378,8 +379,7 @@ func timedCommand(timeout time.Duration, name, dir string, env []string, args ..
 
 	// check whether git was available in the first place
 	if err != nil {
-		_, err := exec.LookPath(name)
-		if err != nil {
+		if !isBinaryInstalled(name) {
 			return "", "", ErrGitNotAvailable
```

Comment:
we allow specifying a path to the git binary in gitserver (which I admit, is not that useful). That's what the other constructor enables.